pennylane_cirq/ops.py

```diff
@@ -49,102 +49,103 @@
-    Controlled-controlled-not gate.
+# class CCNOT(Operation):
+#     r"""CCNOT(wires)
+#     Controlled-controlled-not gate.
 
-    .. math::
+#     .. math::
 
-        CCNOT = \begin{bmatrix}
-            1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
-            0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\
-            0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\
-            0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\
-            0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 \\
-            0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 \\
-            0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 \\
-            0 & 0 & 0 & 0 & 0 & 0 & 1 & 0
-        \end{bmatrix}
+#         CCNOT = \begin{bmatrix}
+#             1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+#             0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\
+#             0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\
+#             0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\
+#             0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 \\
+#             0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 \\
+#             0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 \\
+#             0 & 0 & 0 & 0 & 0 & 0 & 1 & 0
+#         \end{bmatrix}
 
-    **Details:**
+#     **Details:**
 
-    * Number of wires: 3
-    * Number of parameters: 0
+#     * Number of wires: 3
+#     * Number of parameters: 0
 
-    Args:
-        wires (int): the subsystem the gate acts on
-    """
-    num_params = 0
-    num_wires = 3
-    par_domain = None
+#     Args:
+#         wires (int): the subsystem the gate acts on
+#     """
+#     num_params = 0
+#     num_wires = 3
+#     par_domain = None
 
 
-class CSWAP(Operation):
-    r"""CSWAP(wires)
-    Controlled-swap gate.
+# class CSWAP(Operation):
+#     r"""CSWAP(wires)
+#     Controlled-swap gate.
 
-    .. math::
+#     .. math::
 
-        CSWAP = \begin{bmatrix}
-            1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
-             0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\
-             0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\
-             0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\
-             0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 \\
-             0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 \\
-             0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 \\
-             0 & 0 & 0 & 0 & 0 & 0 & 0 & 1
-        \end{bmatrix}
+#         CSWAP = \begin{bmatrix}
+#             1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+#              0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\
+#              0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\
+#              0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\
+#              0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 \\
+#              0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 \\
+#              0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 \\
+#              0 & 0 & 0 & 0 & 0 & 0 & 0 & 1
+#         \end{bmatrix}
 
-    **Details:**
+#     **Details:**
 
-    * Number of wires: 3
-    * Number of parameters: 0
+#     * Number of wires: 3
+#     * Number of parameters: 0
 
-    Args:
-        wires (int): the subsystem the gate acts on
-    """
-    num_params = 0
-    num_wires = 3
-    par_domain = None
+#     Args:
+#         wires (int): the subsystem the gate acts on
+#     """
+#     num_params = 0
+#     num_wires = 3
+#     par_domain = None
```

Comment:
How come this is commented?

Reply:
Those were the extra-ops from the plugin-template. I wanted to keep them as a template for when I add additional operations for Cirq but if they are not commented the coverage goes down.